src/background/hypothesis-chrome-extension.js

```diff
@@ -57,6 +54,9 @@ export default function HypothesisChromeExtension({
     isAllowedFileSchemeAccess,
   });
 
+  /** @type{Map<number, string>} */
```

Comment:
```suggestion
  /** @type {Map<number, string>} */
```

Small stylistic nit. Prettier unfortunately can't format JSDoc for us.